lighthouse-core/test/audits/redirects-test.js

```diff
@@ -107,10 +107,11 @@ const FAILING_CLIENTSIDE = [
 describe('Performance: Redirects audit', () => {
   const mockArtifacts = (networkRecords, finalUrl) => {
     const devtoolsLog = networkRecordsToDevtoolsLog(networkRecords);
+    const frameUrl = networkRecords[0].url;
 
     return {
       GatherContext: {gatherMode: 'navigation'},
-      traces: {defaultPass: createTestTrace({traceEnd: 5000})},
+      traces: {defaultPass: createTestTrace({frameUrl, traceEnd: 5000})},
```

Comment:
a few of these tests were accidentally relying on the fact that the redirects audit was only using navStart events for the redirect chain [if they have a `isLoadingMainFrame` property](https://github.com/GoogleChrome/lighthouse/blob/f2db965f75a4f9d2a84614a3cd5a1466a41e99bc/lighthouse-core/audits/redirects.js#L65), and `create-test-trace` wasn't setting that property. The generated navStart event now has that property and these tests now set the navStart `documentLoaderURL` properly